### DISCUSSION TITLE
Profiling in production code

### DISCUSSION THREAD

**Discussion by AlithAnar** [rpivo/react-automation-profiler#67] [Ideas]
Hi, basically react-automation-profiler is a proxy of React.Profiler which is running well on a development code. If we would like to run react-automation-profiler on the hosted prod react app I'd add a README note mentioning that potential developer needs to follow the [rules](https://reactjs.org/docs/profiler.html)

```
Note:

Profiling adds some additional overhead, so it is disabled in the production build.

To opt into production profiling, React provides a special production build with profiling enabled. Read more about how to use this build at fb.me/react-profiling
```

**Comment by rpivo** [comment]
Great point -- will add a note about this in the readme.
